src/timeMachine/components/TagSelector.tsx

```diff
@@ -97,29 +92,12 @@ class TagSelector extends PureComponent<Props> {
     return (
       isFlagEnabled('newQueryBuilder') &&
       index === 0 &&
-      selectedKey === '_measurement' &&
+      (selectedKey === '' || selectedKey === '_measurement') &&
```

Comment:
The core of what this is doing here is determining whether the first tag selector (index === 0) is the empty or the measurement tag. The reason being that the measurement tag isn't populate as the selected field until the data is returned from the API. The problem this introduces for us is that when a user is loading data from a bucket with a lot of data, the loading state ends up in this weird in-between